src/lib/config/defaultConfig.js

```diff
@@ -41,6 +41,7 @@ export const APP_CONFIG = {
   STATIC_PAGES: [
     { name: 'about', route: '/about', apiURL: '1' },
     { name: 'contact', route: '/contact', apiURL: '2' },
+    { name: 'search-guide', route: '/guide/search', apiURL: '3' },
```

Comment:
how do we make sure this page is available ? did you test it with invenio-app-ils backend ?